### DISCUSSION TITLE
Skaling a browser source should change the webpage's DPI scale

### DISCUSSION THREAD

**Discussion by leochesscom** [obsproject/obs-studio#7791] [Development Questions] [answered]
Skaling a browser source should change the DPI scale setting of the webpage. (the thing that happens when you hit "Ctrl + Plus" and "Ctrl + Minus")

This way a browser source always stays sharp. And users won't run into [this problem](https://obsproject.com/forum/threads/browsersource-is-blurry.116810/) anymore.

+ Option to disable it and set the dpi scale manually.

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a feature request, and we are not currently accepting feature requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [ideas page](https://ideas.obsproject.com) for feature submissions.

Thank you!
